source/PlayState.hx

```diff
@@ -1,35 +1,58 @@
 package;
 
+import flixel.FlxG;
 import flixel.FlxState;
-import flixel.text.FlxText;
-import flixel.util.FlxColor;
+import flixel.group.FlxGroup;
+//import flixel.text.FlxText;
+//import flixel.util.FlxColor;
 
 class PlayState extends FlxState
 {
+	public var ground:FlxGroup;
+	public var moveHero:TopDownHero;
+	public var walltime:PlatformerHero;
+	
 	override public function create():Void
 	{
 		super.create();
-		var text = new FlxText(0, 0, 0, "hello world", 64);
-		text.screenCenter();	
-		text.setFormat(null , 32, FlxColor.RED, CENTER, SHADOW, FlxColor.PINK);
-		text.bold = true;
-		text.borderSize = 5;
-		text.moves = true;
-		text.velocity.set(50, 25);
+		//var text = new FlxText(0, 0, 0, "hello world", 64);
```

Comment:
This is a general comment that applies to a lot of spots in the code:

Our plan is to merge this into your repository, so cleaning up leftover (or commented out) code is a good idea. You can look back at previous commits if you ever want to see your old code, so I don't think there's too big of a benefit of leaving the unused stuff here.